lib/api/core/plugins/pluginContext.js

```diff
@@ -21,44 +21,38 @@ function PluginContext (kuzzle) {
     this.errors[_.upperFirst(name)] = constructor;
   });
 
-  /*
-   lazy-loading accessors, accessible only to plugins loaded by
-   a Kuzzle Server instance
-    */
-  if (kuzzle.isServer) {
-    Object.defineProperty(this.accessors, 'passport', {
-      enumerable: true,
-      get: function () {
-        return {
-          use: kuzzle.passport.use
-        };
-      }
-    });
+  Object.defineProperty(this.accessors, 'passport', {
```

Comment:
I also wanted to add a documentation link to my comment, but it seems I forgot to mention this point about worker plugins when I rewrote the documentation. :fearful: 

I'll submit a PR on the Kuzzle Guide documentation ASAP.